appconfig

```diff
@@ -0,0 +1,4 @@
+application_name=fsm-extension-sample
+application_version=0.1.0
+helm_chart_version=0.1.0
+docker_registry=sapfsm
```

Comment:
The docker registry is public on Docker hub?

Reply:
it is actually a organization on public dockerhub, refer https://hub.docker.com/orgs/sapfsm/repositories
maybe I should change it to docker.io/sapfsm, to make the it more clear

Reply:
Ok